<?xml version="1.0"?>
<launch>
    <node pkg="lio_pkg" type="lio_eskf_ndt_online" name="lio_eskf_ndt_online" output="screen">
        <!-- 加载主要的配置文件 -->
        <rosparam command="load" file="$(find lio_pkg)/config/velodyne_nclt.yaml" />
        
        <!-- 设置运行参数 -->
        <param name="bag_path" type="string" value="/dataset/nclt/nclt9.bag" />
        <param name="dataset_type" type="string" value="NCLT" />
        <param name="config" type="string" value="$(find lio_pkg)/config/velodyne_nclt.yaml" />
        <!-- <param name="display_map" type="bool" value="true" /> -->
    </node>

    <!-- 启动 Rviz -->
    <node pkg="rviz" type="rviz" name="rviz" 
          args="-d $(find lio_pkg)/rviz/show.rviz" 
          output="screen"/>
</launch>